src/jit/codegeninterface.h

```diff
@@ -75,12 +75,12 @@ class CodeGenInterface
     //-------------------------------------------------------------------------
     //  The following property indicates whether to align loops.
     //  (Used to avoid effects of loop alignment when diagnosing perf issues.)
-    __declspec(property(get = doAlignLoops, put = setAlignLoops)) bool genAlignLoops;
-    bool doAlignLoops()
+
+    bool GetAlignLoops()
```

Comment:
Please return it back to `DoAlignLoops'.

Reply:
still need a better name IMO, doAlignLoops() sounds like doing something whereas you are seeking for a bool value here. Something with IsXYZ?

Reply:
`ShouldAlignLoops`?

Reply:
WFM. Consensus?

Reply:
`ShouldAlignLoops` sounds better than `GetAlignLoops` or  `DoAlignLoops`, thanks @jakobbotsch.